RepeaterTask.cpp

```diff
@@ -85,7 +85,7 @@ void RepeaterTask::InterruptRxData()
                 isProtocolMsgReady = true;
 
                 // Notify the protocol task
```

Comment:
Nit: fix comment

Reply:
this what you meant?

`// Notify the repeater task`